keras_cv/layers/preprocessing/random_cutout.py

```diff
@@ -14,31 +14,36 @@
 import tensorflow as tf
 
 from keras_cv.utils import fill_utils
+from keras_cv.utils import preprocessing
 
 
 @tf.keras.utils.register_keras_serializable(package="keras_cv")
 class RandomCutout(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
     """Randomly cut out rectangles from images and fill them.
 
     Args:
-        height_factor: One of:
-            - a positive float representing a fraction of image height
-            - an integer representing an absolute height
-            - a tuple of size 2, representing lower and upper bound for height. For
-            example, `height_factor=(0.2, 0.3)` results in a height randomly picked
-            in the range `[20% of image height, 30% of image height]`.
-            `height_factor=(32, 64)` results in a height picked in the range
-            [32, 64]. `height_factor=0.2` results in a height of [0%, 20%] of image
-            height, and `height_factor=32` results in a height between [0, 32].
-        width_factor: One of:
-            - a positive float representing a fraction of image width
-            - an integer representing an absolute width
-            - a tuple of size 2, representing lower and upper bound for width. For
-            example, `width_factor=(0.2, 0.3)` results in a width randomly picked
-            in the range `[20% of image width, 30% of image width]`.
-            `width_factor=(32, 64)` results in a width picked in the range
-            [32, 64]. `width_factor=0.2` results in a width of [0%, 20%] of image
-            width, and `width_factor=32` results in a width between [0, 32].
+        height_factor: A tuple of two floats, a single float or a
+            `keras_cv.core.FactorSampler`.  `height_factor` controls the size of the
+            cutouts. `height_factor=0.0` means the rectangle will be of size 0% of the
+            image height, `height_factor=0.1` means the rectangle will have a size of
+            10% of the image height, and so forth.
+
```

Comment:
Argument descriptions cannot include blank lines.

Reply:
Done - fixed in all files